aldryn_faq/urls.py

```diff
@@ -10,6 +10,16 @@
 urlpatterns = patterns('',
     url(r'^$', FaqByCategoryListView.as_view(), name='faq-category-list'),
     url(
+        r'^(?P<category_pk>[0-9]+)-(?P<category_slug>[-\w]+)/$',
```

Comment:
I don't see  a need for it, it's already handled correctly.
Would also break backwards compatibility.